### PR TITLE
feat: add certificates stats

### PR DESCRIPTION
**Pull request by andrey-canon**
## Description

This updates frontend essentials to include the new certificates metric https://edunext.atlassian.net/browse/FUTUREX-489

## Testing instructions
Just go to /eox-nelp/stats/tenant/  and include the show_certificates query param if you want to hide the metric

includes the `CERTIFICATE_IMAGE_URL` setting into the `STATS_SETTINGS` to modify the logo image 

## Results

image

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -15,7 +15,7 @@
   "license": "ISC",
   "homepage": "https://bitbucket.org/edunext/eox-nelp#readme",
   "dependencies": {
-    "@edunext/frontend-essentials": "^1.6.1",
+    "@edunext/frontend-essentials": "^1.7.0-rc",
```
Comment (johanv26) [inline package.json:18]: I think this will be updated after all process is completed. I mean essentials bump?

**Review by johanv26 — APPROVED** [review]
LGTM. Working with the desired behavior. 
Peek 2023-09-14 16-06

File: package.json (hunk 1 above)
Comment (andrey-canon) [reply package.json]: done

Comment (andrey-canon) [reply package.json]: done

**Pull request merged**